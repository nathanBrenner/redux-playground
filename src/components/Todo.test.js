import React from "react";
import Todo from "./Todo";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

describe("Todo", () => {
  test("renders without crashing", () => {
    const onClick = () => true;
    const completed = false;
    const text = "do something";
    const props = { onClick, completed, text };
    shallow(<Todo {...props} />);
  });

  test("snapshot", () => {
    const onClick = () => true;
    const completed = false;
    const text = "do something";
    const props = { onClick, completed, text };
    const component = renderer.create(<Todo {...props} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
