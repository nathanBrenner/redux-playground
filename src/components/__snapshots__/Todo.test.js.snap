// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo snapshot 1`] = `
<li
  onClick={[Function]}
  style={
    Object {
      "textDecoration": "none",
    }
  }
>
  do something
</li>
`;
